lib/plugins/package/lib/packageService.js

```diff
@@ -52,9 +52,12 @@ module.exports = {
   packageAll() {
     const exclude = this.getExcludes();
     const include = this.getIncludes();
-    const zipFileName = this.provider.naming.getServiceArtifactName();
+    const zipFileName = `${this.serverless.service.service}.zip`;
```

Comment:
This was in the naming namespace. But the problem is, that the topmost package command does not have knowledge nor access to the current provider. That's why we changed it here.
A possible improvement would be to create a provider-independent naming module that can be accessed here - but that will only increase debt as long as only one value/function is stored in there.